### PR TITLE
[@riderize/passport-strava-oauth2] Remove, bundles its own types

### PR DESCRIPTION
**Pull request by BendingBender**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If removing a declaration:
- [ ] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [x] Delete the package's directory.
- [x] Add it to `notNeededPackages.json`.

### CODE REVIEW

File: notNeededPackages.json [hunk 1]
```diff
@@ -3738,6 +3738,10 @@
             "libraryName": "rgb-hex",
             "asOfVersion": "3.0.0"
         },
+        "riderize__passport-strava-oauth2": {
+            "libraryName": "@riderize/passport-strava-oauth2",
+            "asOfVersion": "1.1.1"
```
Comment (ImRodry) [inline notNeededPackages.json:3743]: Added on commit https://github.com/auth0-community/auth0-socketio-jwt/commit/72f3846fcf8b9c5e8d3d0850806464152292a02d which belongs to v1.0.9
```suggestion
            "asOfVersion": "1.0.9"
```

Comment (ImRodry) [inline notNeededPackages.json:3743]: Added on commit https://github.com/auth0-community/auth0-socketio-jwt/commit/72f3846fcf8b9c5e8d3d0850806464152292a02d which belongs to v1.0.9
Changelog for v1.0.9: https://github.com/Riderize/passport-strava-oauth2/releases/tag/1.0.9
```suggestion
            "asOfVersion": "1.0.9"
```

**Review by ImRodry — CHANGES_REQUESTED** [review]

File: notNeededPackages.json (hunk 1 above)
Comment (BendingBender) [reply notNeededPackages.json:3743]: This is not possible, there is already a `@types/riderize__passport-strava-oauth2@1.1.0` published on npm. Thus, I simply took the next version.

Comment (ImRodry) [reply notNeededPackages.json:3743]: This is supposed to be the number of the version that added type declarations for this package, it's not the version of the `@types` package

Comment (BendingBender) [reply notNeededPackages.json:3743]: The checks won't pass. You can try it out yourself with `npm run test-all`. This is actually supposed to be the version of the `@types` package that will be published with the obsoleted warning.

**Pull request merged**